pkg/cli/cli.go

```diff
@@ -8,8 +8,13 @@ package cli
 
```

Comment:
This should probably mostly be in `trace.go` or something like that, to keep the file small